### ISSUE TITLE
Having problems using autotrain

### ISSUE THREAD

**Issue by DeathlyRoyal** [huggingface/autotrain-advanced#896]
So Currently im trying to train the ai locally, its basic text generation ai
Using model: meta-llama/Llama-3.1-8B-Instruct
Parameters are the basic:
{
  "chat_template": "chatml",
  "distributed_backend": "ddp",
  "mixed_precision": "fp16",
  "optimizer": "adamw_torch",
  "peft": "true",
  "scheduler": "linear",
  "unsloth": "false",
  "batch_size": "2",
  "block_size": "1024",
  "epochs": "3",
  "gradient_accumulation": "4",
  "lr": "0.00003",
  "model_max_length": "2048",
  "target_modules": "all-linear"
}

I dont know much about these so i used ChatGPT for help, i structured my training data like this:
{"instruction": "Describe where you are going. Keep the answer short.", "output": "You tell me, i have no idea."}

But since that didn't work ChatGPT suggested i restructure it so i can run it with my current Column Mapping to
{"text": "### Instruction:\nDescribe where you are going. Keep the answer short.\n### Response:\nYou tell me, i have no idea.", "instruction": "Describe where you are going. Keep the answer short.", "output": "You tell me, i have no idea."}

I have around 100 lines like these, im also assuming it is not structured correctly as i don't know how a good file is supposed to be structures.
Also its .jsonl file

Once i do try to train it the top left is filled with Red
Running job PID(s): 4430
I dont know what this means but since its red i dont think it is good.

And after the crash the next error appears:
INFO:     127.0.0.1:57320 - "GET /ui/is_model_training HTTP/1.1" 401 Unauthorized
ERROR    | 2025-06-08 15:26:26 | autotrain.app.utils:token_verification:138 - Failed to request /api/whoami-v2 - 429
ERROR    | 2025-06-08 15:26:26 | autotrain.app.ui_routes:user_authentication:343 - Failed to verify token: Invalid token (/api/whoami-v2). Please login with a write token.

Using huggingface-cli whoami gives me a big error in form of a website page that says i was rate limited and that i should buy an upgraded enterprise account so i can ask for support.

Column Mapping is just "text" i don't really know what to type in here, this is most likely the problem.
And again i dont even know if i structured the data correctly.
All i want is to change its personality, like to make it answer differently.
